cypress/e2e/tests/login.test.ts

```diff
@@ -26,5 +26,7 @@ describe("Log In", () => {
 
         // Assert that home page has loaded after login
         cy.get("h1").should("contain", "Application inventory");
+        // Clean up any previous existing applications
+        deleteApplicationTableRows();
```

Comment:
We shouldn't do this on each login, this might break some tests that require keeping data and re-login, RBAC for example

Reply:
That was committed by mistake . removed .Thanks for noticing.